spec/features/navigation_spec.rb

```diff
@@ -42,6 +60,35 @@
     end
 
     describe 'I see a nav bar where I can link to' do
+      it 'the welcome page' do
+        visit user_dashboard_path
+
+        within 'nav' do
+          click_link 'Home'
+        end
+
+        expect(current_path).to eq(root_path)
+      end
+
+      it 'the profile page' do
+        visit user_dashboard_path
+
+        within 'nav' do
+          click_link 'Profile'
+        end
+
+        expect(current_path).to eq('/users/11')
+      end
+
+      it 'the dashboard page' do
+        visit root_path
+
+        within 'nav' do
+          click_link 'Dashboard'
+        end
+
+        expect(current_path).to eq(user_dashboard_path)
+      end
```

Comment:
Just beefing up the testing for the nav bar.  More testing is needed to get coverage up to 100%